core/src/main/java/site/ycsb/workloads/CoreWorkload.java

```diff
@@ -83,6 +83,17 @@ public class CoreWorkload extends Workload {
 
   protected String table;
 
+  /**
+   * The name of the property for the number of test tables.
+   * Table names will be formed from TABLENAME_PROPERTY_DEFAULT value with adding index at the end.
+   */
+  public static final String TABLE_COUNT_PROPERTY = "tablecount";
```

Comment:
These two fields are meant to be used by the Ignite 3 benchmarks only. Why not to create them in IgniteAbstractClient instead of in the CoreWorkload? I.e. follow the same approach as with `disableFsync` and `useColumnar`.